src/test/java/com/google/sps/AvailabilityTest.java

```diff
@@ -14,17 +14,37 @@
 
 package com.google.sps;
 
+import java.util.List;
+import java.util.Arrays;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import com.google.sps.data.TimeRange;
+import com.google.gson.Gson;
 
 @RunWith(JUnit4.class)
 public final class AvailabilityTest {
+    private static final int TIME_0800AM = TimeRange.getTimeInMinutes(8, 00);
+    private static final int TIME_0900AM = TimeRange.getTimeInMinutes(9, 00);
+    private static final int TIME_1000AM = TimeRange.getTimeInMinutes(10, 00);
+    private static final int TIME_1100AM = TimeRange.getTimeInMinutes(11, 00);
+    private static final int TIME_1200AM = TimeRange.getTimeInMinutes(12, 00);
 
     @Test
-    public void noFailTest() {
-        Assert.assertEquals(true, true);
+    public void testConvertToJsonUsingGson() {
```

Comment:
I think you should maybe add a test for your doPost method and make sure it returns the correct response (like the correct tutor's timeslots for the id parameter).

Reply:
Yes, that's important. My newest commit to this PR addresses that test.

Reply:
Awesome!